src/if-merge/util/helpers.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path';
+
+import {getFileName, getYamlFiles} from '../../common/util/fs';
+import {Context} from '../../common/types/manifest';
+import {load} from '../../common/lib/load';
+
+import {ExportYaml} from '../../if-run/builtins/export-yaml';
+
+import {IFMergeArgs} from '../types/process-args';
+
+/**
+ * Merges the given manifests in the one file.
+ */
+export const mergeManifests = async (commandArgs: IFMergeArgs) => {
```

Comment:
this function can be simplified, and please make sure variable names are clearly describing what they contain